tests/index.js

```diff
@@ -611,6 +611,14 @@ t('Transform nested json in arrays', async() => {
   return ['aBcD', (await sql`select '[{"a_b":1},{"c_d":2}]'::jsonb as x`)[0].x.map(Object.keys).join('')]
 })
 
+t('Transform null json in arrays', async() => {
+  const sql = postgres({
+    ...options,
+    transform: postgres.camel
+  })
+  return [null, (await sql`select '[{"a_b":null},{"c_d":null}]'::jsonb as x`)[0].x.map(Object.keys).join('')]
```

Comment:
I'm not sure this condition does what you think it does (I imagine this test will fail)

setting a jsonb value to an array has already been done in the previous test - your new test should make sure that json and jsonb values which are both not arrays and also not objects do not get transformed (and do not cause this crash that you experienced)

Reply:
The commit
https://github.com/porsager/postgres/pull/510/commits/b8242f357053fa006bc2af394cb55db2bcc7fc05

Reply:
@porsager, What do you think about the PR? 
I think it is ready for you